src/arch/simddetect.h

```diff
@@ -22,7 +22,7 @@
 namespace tesseract {
 
 // Function pointer for best calculation of dot product.
-typedef double (*DotProductFunction)(const double* u, const double* v, int n);
+using DotProductFunction = double (*)(const double*, const double*, int);
```

Comment:
The fix drops the var names, is it ok?

Reply:
Technically it is fine, but nevertheless that is a side effect of `clang-tidy` which I did not intend. Maybe `clang-tidy` can be told to keep the names for the function parameters, or I have to add them manually again if we prefer to have those names.

Personally I want the names at least in public API header files.

Reply:
Yes, I agree.